docs/openapi/components/schemas/common/Organization.yml

```diff
@@ -115,20 +101,72 @@ properties:
     $linkedData:
       term: iataCarrierCode
       '@id': https://onerecord.iata.org/cargo/Company#airlineCode
+  organizationType:
+    title: Organization Type
+    description: Type of organization (e.g. cold storage, meat processing).
+    type: string
+    $linkedData:
+      term: organizationType
+      '@id': https://w3id.org/traceability#organizationType
+  place:
+    title: Place
+    description: Information on organization place.
+    $ref: ./Place.yml
+    $linkedData:
+      term: place
+      '@id': https://schema.org/Place
+  gap:
+    title: GAP Certifications
+    description: List of GAP certifications and links.
+    type: string
+    $linkedData:
+      term: gap
+      '@id': https://w3id.org/traceability#gap
+  generalCerts:
+    title: General Certifications
+    description: List of general certificates (such as export certificates, etc.) possessed by the organization.
+    type: string
+    $linkedData:
+      term: generalCerts
+      '@id': https://w3id.org/traceability#generalCerts
+  trustedBy:
```

Comment:
How do we expect this to work?
is this a group of organizations that say they trust this organization? what happens if they no longer trust this organization? will this organization need to reissue its VC?

Reply:
yeah, interested to understand this

Reply:
This will likely go another route - common scenario we encounter is an attestation by a 3rd party that they trust some org to have X internal credential - i think in our case AgInspection or another more general cert that references the org is better than embedding in the org.

@rhofvendahl i suggest removing generalCerts and trustedBy for now - and perhaps add a general attestation credential - e.g. org A says the following claims about org B